content/influxdb/cloud-iox/sql-data-types.md

```diff
@@ -0,0 +1,133 @@
+---
+title: SQL data types
+description: >
+  SQL data types
+menu:
+  influxdb_cloud_iox:
+    name: SQL data types
+    parent: Query data with SQL
+weight: 105
+---
+
+InfluxDB Cloud backed by InfluxDB IOx uses the Apache Arrow DataFusion implementation of SQL. Data types define the type of value that can be stored in table column. In InfluxDB's SQL implementation, a `measurement` is a table, and columns are comprised of `tags`, `fields` and `timestamps`. 
+
+DataFusion uses the Arrow type system for query execution. Data types stored in InfluxDB's storage engine are mapped to SQL data types according to the following tables. 
+
+{{% note %}}
+When performing casting operations, cast to the **name** of the data type, not the actual data type.  Names are not case sensitive. 
+
+```sql
+--Examples
+SELECT
+'99'::BIGINT
+'2019-09-18T00:00:00Z'::timestamp
+```
```

Comment:
~~~suggestion
When performing casting operations, cast to the **name** of the data type, not the actual data type. 
Names are _case-insensitive_. For example:

```sql
SELECT
  '99'::BIGINT, 
  '2019-09-18T00:00:00Z'::timestamp
```
~~~